noisemodelling-jdbc/src/main/java/org/noise_planet/noisemodelling/jdbc/NoiseMapByReceiverMaker.java

```diff
@@ -58,8 +57,15 @@ public PointNoiseMap(String buildingsTableName, String sourcesTableName, String
      * to profile the execution of the simulation
      * @return Instance of ProfilerThread or null
      */
-    public ProfilerThread getProfilerThread() {
+    /*public ProfilerThread getProfilerThread() {
```

Comment:
If this function is never used and you want to remove it, delete the code not comment it out.
Except for Java Getter/Setter, don't remove the functions even if it is not used (will be used outside of the library)